services/src/main/java/org/exoplatform/addons/gamification/entities/domain/effective/GamificationActionsHistory.java

```diff
@@ -27,332 +27,348 @@
 import org.exoplatform.addons.gamification.IdentityType;
 import org.exoplatform.addons.gamification.entities.domain.configuration.AbstractAuditingEntity;
 import org.exoplatform.addons.gamification.entities.domain.configuration.DomainEntity;
+import org.exoplatform.addons.gamification.entities.domain.configuration.RuleEntity;
 import org.exoplatform.commons.api.persistence.ExoEntity;
 
 @ExoEntity
 @Entity(name = "GamificationActionsHistory")
 @Table(name = "GAMIFICATION_ACTIONS_HISTORY")
 @NamedQueries({
-    @NamedQuery(name = "GamificationActionsHistory.findAllActionsHistory", query = "SELECT" +
```

Comment:
Could you check that you use the correct formatting configuration ? https://developer.exoplatform.org/#id-ide-settings-to-work-on-exo-platform-projects

Reply:
done!